src/compass/s1_cslc_qa.py

```diff
@@ -0,0 +1,144 @@
+
+            # compute stats for real and complex
+            stat_obj = isce3.math.StatsRealImagFloat32(ds[()])
+
+            # create dict for current polarization
+            self.stats_dict[pol] = {}
+            pol_dict = self.stats_dict[pol]
+
+            # write stats to HDF5
+            for real_imag, cstat_member in zip(['real', 'imag'],
+                                               [stat_obj.real, stat_obj.imag]):
+                # create dict to store real/imaginary stat items
+                pol_dict[real_imag] = {}
+
+                vals = [cstat_member.mean, cstat_member.min,
+                        cstat_member.max, cstat_member.sample_stddev]
+                # write stats to HDF5
+                for val_name, val in zip(self.stat_names, vals):
+                    pol_dict[real_imag][val_name] = val
+
+
+    def compute_correction_stats(self, cslc_h5py_root):
+        '''
+        Compute correction stats
+
+        Parameters
+        ----------
+        cslc_h5py_root: h5py.File
+            Root of CSLC HDF5
+        '''
+        # compute and save corrections stats
+        corrections = ['bistatic_delay', 'geometry_steering_doppler',
+                       'azimuth_fm_rate_mismatch']
+
+        # compute stats and write to hdf5
+        self.stats_dict['corrections'] = {}
+        corrections_dict = self.stats_dict['corrections']
+        for correction in corrections:
+            # create dict for current correction
+            corrections_dict[correction] = {}
+            correction_dict = corrections_dict[correction]
+
+            # get dataset and compute stats according to dtype
+            src_path = f'{ROOT_PATH}/corrections/{correction}'
+            ds = cslc_h5py_root[src_path]
+            stat_obj = isce3.math.StatsFloat32(ds[()].astype(np.float32))
+
+            # write stats to HDF5
+            vals = [stat_obj.mean, stat_obj.min, stat_obj.max]
+            for val_name, val in zip(self.stat_names, vals):
+                correction_dict[val_name] = val
+
+
+    def raster_pixel_classification(self):
+        '''
+        Place holder for populating classification of geocoded pixel types
+        '''
+        self.classification_count_dict['topo'] = {}
+        topo_dict = self.classification_count_dict['topo']
+        topo_dict['layover_percent'] = 0
+        topo_dict['shadow_percent'] = 0
+        topo_dict['combined_percent'] = 0
+        self.classification_count_dict['percent_land_pixels'] = 0
+        self.classification_count_dict['percent_valid_pixels'] = 0
+
+
+    def populate_rfi_dict(self):
+        '''
+        Place holder for populating SAFE RFI information
+        '''
+        self.rfi_dict['is_rfi_info_available'] = True
+        # Follow key/values only assigned if RFI info is avaiable
+        self.rfi_dict['rfi_mitigation_performed'] = True
+        self.rfi_dict['rfi_mitigation_domain'] = True
+        self.rfi_dict['rfi_burst_report'] = True
```

Comment:
@vbrancat @seongsujeong Do we want to include descriptions in addition to values like in the metadata?

Current JSON output (just value)
```json
    "rfi_info": {
        "is_rfi_info_available": true,
        ...
    }
```

JSON with description and value
```json
    "rfi_info": {
        "is_rfi_info_available":  {
            "value": true,
            "description": "Boolean variable specifying availability of RFI information"
        },
        ...
    }
```

Reply:
@scottstanie Forgot to tag you. Do you have an opinion on this?

Reply:
I think in general that it's good to have the description- it'll add minimal overhead, and, for instance, I don't think I could guess what `rfi_mitigation_domain` is just from the name.

Reply:
I agree with @scottstanie. I think it's better to add the description.